src/main/java/net/minecraftforge/fml/common/FMLCommonHandler.java

```diff
@@ -663,7 +663,7 @@ public void processWindowMessages()
      */
     public void exitJava(int exitCode, boolean hardExit)
     {
-        FMLLog.log.info("Java has been asked to exit (code {}) by {}.", exitCode, Thread.currentThread().getStackTrace()[1]);
+        FMLLog.log.info("Java has been asked to exit (code {}) by {}.", exitCode, Thread.currentThread().getStackTrace()[2]);
         if (hardExit)
         {
             FMLLog.log.info("This is an abortive exit and could cause world corruption or other things");
```

Comment:
While you're here, could you make this `warn` instead of `info`?

Reply:
Honestly, its not like its spammed a lot, i'd be fine just posting the entire stack.

Reply:
That can (currently) be enabled with a VM argument (a few lines down). Do you want that as the new default, or is the current behavior (telling the user how to get a full stack in the message) ok?

Reply:
Yeah I think this should just show the whole stack. Maybe remove the top two lines of the stack so we get the best of both of these things.